yt/utilities/io_handler.py

```diff
@@ -60,6 +60,22 @@ def push(self, grid, field, data):
             raise ValueError
         self.queue[grid][field] = data
 
+    @property
```

Comment:
My only comment here is that I'd love it if it was more general (as this relates to the use case that @chunyen-chen was talking about). But if that is not easy to do in this PR we can just submit another one.

Reply:
well my thinking was this `_particle_dtypes` dictionary would be a general, front-end dependent dictionary mapping types to fields. Any ideas on how to make it more general/useable? Looking at it now after some months, I do think it could make sense to instead have this field-dtype mapping defined over in the `FieldInfoContainer`, but that would still require each frontend to over-ride or add to it as needed.